examples/CMakeLists.txt

```diff
@@ -20,4 +20,4 @@ if (NOT GTSAM_USE_BOOST_FEATURES)
       )
 endif()
 
-gtsamAddExamplesGlob("*.cpp" "${excluded_examples}" "gtsam;gtsam_unstable;${Boost_PROGRAM_OPTIONS_LIBRARY}")
+gtsamAddExamplesGlob("*.cpp" "${excluded_examples}" "gtsam;${Boost_PROGRAM_OPTIONS_LIBRARY}")
```

Comment:
There should be an if-else condition here based on `GTSAM_BUILD_UNSTABLE`.

Reply:
Here's what I would do:
```cmake
if (GTSAM_BUILD_UNSTABLE)
  set(GTSAM_UNSTABLE_LIB "gtsam_unstable")
else()
  set(GTSAM_UNSTABLE_LIB "")
endif (GTSAM_BUILD_UNSTABLE)
gtsamAddExamplesGlob("*.cpp" "${excluded_examples}" "gtsam;${GTSAM_UNSTABLE_LIB};${Boost_PROGRAM_OPTIONS_LIBRARY}")
```

Reply:
Frank and I discussed and we moved examples that depend on `gtsam_unstable` to `gtsam_unstable/examples`.